varfish_db_downloader/wget.py

```
@@ -11,6 +12,7 @@
 import typing
 import urllib.parse
 import zlib
+from urllib.parse import urlparse
```

Ruff (0.11.9)

`urllib.parse.urlparse` imported but unused

Remove unused import: `urllib.parse.urlparse`

(F401)